rs/replicated_state/src/canister_state.rs

```diff
@@ -120,9 +123,11 @@ pub struct CanisterState {
     /// an actual wasm module. A valid canister is not required to contain a
     /// Wasm module. Canisters without Wasm modules can exist as a store of
     /// ICP; temporarily when they are being upgraded, etc.
+    #[validate_eq(Ignore)]
```

Comment:
I think it would be good to go a bit deeper here. A lot inside should be ignored, but e.g.  `Memory::size` and `last_executed_round` don't need to.